zulip_bots/zulip_bots/bots/monkeytest/fixtures/test_bot.json

```diff
@@ -0,0 +1,17 @@
+{
+  "request":{
+    "api_url":"https://monkeytest.it/test",
+    "method":"POST",
+    "params":{
+      "url":"http://www.google.com",
+      "secret":"12345"
+    }
+  },
+  "response":{
+    "results_url":"https://monkeytest.it/test/qwerty123"
+  },
+  "response-headers": {
+    "status": 200,
+    "content-type": "application/json; charset=utf-8"
+  }
+}
```

Comment:
Make sure you include a newline at the end of this file so that little red sign goes away! :)